src/Components/SightingsMap/SightingsMap.js

```diff
@@ -1,11 +1,101 @@
-import React from "react";
+import React, { useState } from "react";
+import "./SightingsMap.scss";
+import {
+  GoogleMap,
+  useJsApiLoader,
+  Marker,
+  InfoWindow,
+} from "@react-google-maps/api";
+import ufo from "../../assets/ufobeam.png";
+import alien from "../../assets/alienmarker.png";
+import lights from "../../assets/skylights.png";
+import { containerStyle, mapCenter, customMap } from "../../assets/mapSetup";
+import { mockSightings } from "../../mockdata";
 
 const SightingsMap = () => {
-  return (
-    <main>
-      <h1>Map here</h1>
-    </main>
-  )
-}
-
-export default SightingsMap;
\ No newline at end of file
+  const [selectedCenter, setSelectedCenter] = useState(null);
+  const [selectedSite, setSelectedSite] = useState(null);
+
+  const { isLoaded } = useJsApiLoader({
+    id: "google-map-script",
+    googleMapsApiKey: `${process.env.REACT_APP_API_KEY}`,
+  });
+
+  const generateIconType = (eventType) => {
+    switch (eventType) {
+      case "sighting":
+        return lights;
+      case "encounter":
+        return alien;
+      case "abduction":
+        return ufo;
+      default:
+        return lights;
+    }
+  };
+
+  const generateMarkers = () => {
+    return mockSightings.map((sighting) => {
```

Comment:
Nice work with the mock sightings, this should make it easy to switch to getting the data from the backend.